addons/medical_engine/XEH_postInit.sqf

```diff
@@ -94,3 +94,19 @@
         };
     };
 }] call CBA_fnc_addEventHandler;
+
+["ace_killed", { // global event
+    params ["_unit"];
+    TRACE_3("unit Killed",_unit,objectParent _unit,local _unit);
+    if (!isNull objectParent _unit && {local objectParent _unit}) exitWith {
```

Comment:
We might be able to switch this to just
```suggestion
    if (local objectParent _unit) exitWith {
```
as `local objNull` seems to always be false, but it's a minuscule gain and less readable IMO.